webdev/nextjs/webdev.sh

```diff
@@ -157,15 +138,15 @@ case "$1" in
         npx next build  || exit $?
         npx next export || exit $?
         cd azion/cells-site-template || exit $?
-        azion-framework-adapter build --config ../kv.json \
+        npx --yes azion-framework-adapter@0.2.0 build --config ../kv.json \
```

Comment:
```suggestion
        npx --yes azion-framework-adapter@0.2 build --config ../kv.json \
```
Dropping the patch number should allow automatic to update the npm package  for bugs.

Reply:
https://github.com/aziontech/azioncli-template/pull/35#discussion_r1001968121